pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -60,7 +60,7 @@ const (
 
 var (
 	// PropagationPolicy defines the propagation policy used when deleting a resource
-	PropagationPolicy = metav1.DeletePropagationForeground
+	PropagationPolicy = metav1.DeletePropagationBackground
```

Comment:
Why replace `foreground` with `background`? Shouldn't we want to make sure all dependents are garbage collected? Will `backgound` cause orphan dependents?

Reply:
`foreground` needs to wait for all dependents to be collected before deleting the owner.

In kind test, it may take too long to collect the dependents. When the next convert is executed, the resources created last time still exist, so the conversion cannot be performed this time.

`background` will delete the owner first, and then the garbage collector will delete its dependencies in the background.